hack/test-utils.sh

```diff
@@ -40,13 +40,14 @@ test_setup() {
   fi
 
   # Start containerd
-  if [ ! -x "$(command -v containerd)" ]; then
```

Comment:
How about just using (which sudo containerd) here? then in the comment below  mention it's being run in sudo...

"containerd is not installed, please run hack/install-deps.sh [and check that containerd is on your sudo path as containerd must be run as root for now]."

Reply:
@Random-Liu  @mikebrow updated to echo containerd path and version to be used, I prefer to resolve the path to containerd.

If we use `sudo  which containerd` in the condition, later it may not be possible to  found it because sudo's `secure_path`  might not be included `/usr/loca/bin` ( fedora does not include `/usr/loca/bin`  in sudo `secure_path`)

Alternatively we can modify the line when containerd is executed for something like : 
```bash
keepalive "sudo PATH=$PATH bash -c 'containerd'" ${RESTART_WAIT_PERIOD} &> ${report_dir}/containerd.log &
```
And add sudo to this conditional as you suggest (because `command` is a shell command it will use the shell PATH env variable and not sudo's `secure_path` ) 
```
if [ ! -x "$(sudo command -v containerd)" ]; then
```

Reply:
prefer your alternative but it's merged.. It's a testutil script not a production one, should be fine :-)